lib/osf-components/addon/components/new-feature-popover/styles.scss

```diff
@@ -0,0 +1,16 @@
+:global(.popover) {
```

Comment:
We should add a local-class to this popover, since this will leak onto other usages of BsPopover.